File: dataset/output-bk/GEMINI_gemini-1.5-pro-001/Task25_GEMINI_gemini-1.5-pro-001.cpp
#include <iostream>
#include <vector>

std::vector<std::string> number(const std::vector<std::string>& lines) {
    std::vector<std::string> numberedLines;
    for (int i = 0; i < lines.size(); ++i) {
        numberedLines.push_back(std::to_string(i + 1) + ": " + lines[i]);
    }
    return numberedLines;
}

int main() {
    std::vector<std::string> input = {"a", "b", "c"};
    std::vector<std::string> result = number(input);
    for (const auto& line : result) {
        std::cout << line << std::endl;
    }
    return 0;
}

File: dataset/output-bk/GEMINI_gemini-1.5-pro-001/Task25_GEMINI_gemini-1.5-pro-001.java
class Solution {
    public static void main(String[] args) {
        String[] input = {"a", "b", "c"};
        String[] result = number(input);
        for (String s : result) {
            System.out.println(s);
        }
    }
    public static String[] number(String[] lines) {
        String[] numberedLines = new String[lines.length];
        for (int i = 0; i < lines.length; i++) {
            numberedLines[i] = (i + 1) + ": " + lines[i];
        }
        return numberedLines;
    }
}